### PR TITLE
docs: add example for custom interpretations

### PR DESCRIPTION
**Pull request by tamasgal**
These are just two tiny examples which show how to pass a custom interpretation of a jagged structure in uproot3 vs uproot4.

Not sure if the cheat sheet is intended to look like this, so please feel free to take whatever you think is useful.

Edit: crossref to #169

### CODE REVIEW

File: docs-sphinx/uproot3-to-4.rst [hunk 1]
```diff
@@ -4,3 +4,79 @@ Uproot 3 → 4 cheat-sheet
...
+                )
+            ),
+            skipbytes=10,
+        )
+    )
+
+This will return an ``awkward0.JaggedArray``:
+
+.. code-block:: python
+    >>> snapshot_hits
```
Comment (jpivarski) [inline docs-sphinx/uproot3-to-4.rst:48]: I'm pretty sure you need a gap between `.. code-block:: python` and the code. reStructuredText is finicky.

File: docs-sphinx/uproot3-to-4.rst [hunk 2]
```diff
@@ -4,3 +4,79 @@ Uproot 3 → 4 cheat-sheet
...
+    <JaggedArray [[<Row 0> <Row 1> <Row 2> ... <Row 50> <Row 51> <Row 52>]  ...  [<Row 849> <Row 850> <Row 851> ... <Row 887> <Row 888> <Row 889>] [<Row 890> <Row 891> <Row 892> ... <Row 920> <Row 921> <Row 922>]] at 0x7f9b8e6c89d0>
+
+    >>> snapshot_hits.dom_id
+    <JaggedArray [[808432835 808432835 808432835 ... 809526097 809526097 809526097]   ...  [808432835 808488997 808488997 ... 809526097 809526097 809544061] [808432835 808432835 808432835 ... 809526097 809526097 809544061]] at 0x7f9bc99b05e0>
+
+uproot4
+~~~~~~~
+
+.. code-block:: python
+    import uproot4 as uproot  # version 4
```
Comment (jpivarski) [inline docs-sphinx/uproot3-to-4.rst:58]: And here.

File: docs-sphinx/uproot3-to-4.rst [hunk 3]
```diff
@@ -4,3 +4,79 @@ Uproot 3 → 4 cheat-sheet
...
+                    ("tot", "u1"),
+                ]
+            ), header_bytes=10,
+        )
+    )
+
+Which will return an ``awkward1.Array``:
+
+.. code-block:: python
+    >>> snapshot_hits
```
Comment (jpivarski) [inline docs-sphinx/uproot3-to-4.rst:81]: And here.

**Review by jpivarski — APPROVED** [review]

File: docs-sphinx/uproot3-to-4.rst (hunk 1 above)
Comment (tamasgal) [reply docs-sphinx/uproot3-to-4.rst:48]: Absolutely right, I always forget about that...

**Pull request closed without merging**